cmd/repl/tablewriter/ascii.go

```diff
@@ -60,29 +62,38 @@ func newASCIITableWriter(bufSize int, w io.Writer) *ASCIITableWriter {
 }
 
 // SetHeader set the table header
-func (s *ASCIITableWriter) SetHeader(header []string) error {
-	if len(header) == 0 {
-		return fmt.Errorf("header columns should not be empty")
+func (t *ASCIITableWriter) SetHeader(head map[string]interface{}) error {
```

Comment:
Maybe name `ascii.go` to `ascii_writer.go` and the struct `ASCIITableWriter` to `ASCIIWriter`,  so the full name would be like `tablewriter.ASCIIWriter`. Same to `ProtobufTableWriter` -> `tablewriter. ProtobufWriter`